### DISCUSSION TITLE
2.0-3.0 Update

### DISCUSSION THREAD

**Discussion by InsanityFromScratch** [ScratchAddons/ScratchAddons#1969] [Ideas] [answered]
When you enable the 2.0-3.0 addon, the forums also become modernized. But I've found an issue.
There's an option to change the color of the header, plus several buttons. But the forum's headers are always green.
I suggest you change this.

**Comment by mxmou** [comment, marked as answer]
There is an issue for this: #1092

I plan to add this as part of #645.
